src/url-replacements.js

```diff
@@ -144,6 +148,82 @@ class UrlReplacements {
     this.set_('SCREEN_HEIGHT', () => {
       return this.win_.screen.height;
     });
+
+    // Returns the time it took to load the whole page. (excludes amp-* elements
+    // that are not rendered by the system yet.)
+    this.set_('PAGE_LOAD_TIME', () => {
```

Comment:
Separated into two tests. Single test was resulting in the first part interfering with the second part. 

The reason for having a single test was to demonstrate that the value is empty initially and then changes to something. I think that part is lost with the test now.